compiler/one-cmds/onecc

```diff
@@ -50,7 +100,7 @@ def _verify_arg(parser, args):
     """verify given arguments"""
     # check if required arguments is given
     if not _utils._is_valid_attr(args, 'config'):
-        parser.error('-C/--config argument is required')
+        parser.error('unrecognized arguments: ' + sys.argv[1])
```

Comment:
What is it for? This line invokes when config option is not given.

Reply:
I fixed the code to make it more understandable. Please see the latest patch.

I used the `nargs='?'` options to make the sub-commands as optional arguments. The optional argument means that the user is not forced to select the corresponding option. User can turn this option on or off whenever they want.

The latest patch still use the optional argument but the way is changed to be easy to understand codes.